src/api/app/views/webui/package/bottom_actions/_request_devel_project_change.html.haml

```diff
@@ -1,4 +1,5 @@
 %li.list-inline-item
-  = link_to('#', class: 'nav-link', data: { toggle: 'modal', target: '#change-devel-request-modal' }) do
-    %i.fas.fa-exchange-alt.text-primary
+%li.nav-item
+  = link_to(new_packages_change_devel_projects_path(package.project, package), class: 'nav-link') do
+    %i.fas.fa-exchange-alt.fa-lg.mr-2
     Request devel project change
```

Comment:
Let's be consistent and capitalize this link properly so it's just like the others.

Reply:
Link capitalized.